### ISSUE TITLE
Wildcard globbing for MSVC builds

### ISSUE THREAD

**Issue by busstoptaktik** [OSGeo/PROJ#712]
proj, cct, cs2cs, gie and friends, do not expand wildcards in the argv list, when compiled with MSVC.

According to [this page](https://docs.microsoft.com/da-dk/cpp/c-language/expanding-wildcard-arguments) it is a matter of linking with `setargv.obj`.

 This appears easy to do in a strictly MSVC setup, and also seems simple to add in makefile.vc, for the nmake based builds. But for the cmake-based, my limited cmake-fu makes leaves me at a loss for where to start.

Any takers? (or pointers)

**Comment by kbevers** [comment]
A quick google search gave me this: https://github.com/Tom94/tev/blob/master/CMakeLists.txt 

So I think you can add something like

```
if (MSVC)
    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} setargv.obj")
endif ()
```

to `CMakelists.txt`

**Comment by busstoptaktik** [comment]
Thanks - will try this (some time after merging #693)

**Comment by busstoptaktik** [comment]
Resolved with PR #714
